client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionEventForm/InteractionEventForm.tsx

```diff
@@ -157,7 +158,8 @@ const InteractionEventForm: React.FC<Props> = (
       [InteractionEventDialogFields.END_TIME]: endTimeToSave,
       [InteractionEventDialogFields.ID]: methods.watch(InteractionEventDialogFields.ID),
       [InteractionEventDialogFields.PLACE_NAME]: name || generatePlacenameByPlaceSubType(placeSubtypeName),
-      [InteractionEventDialogFields.EXTERNALIZATION_APPROVAL]: data === null ? false : data.externalizationApproval,
+      [InteractionEventDialogFields.EXTERNALIZATION_APPROVAL]: 
+        data[InteractionEventDialogFields.EXTERNALIZATION_APPROVAL] === null ? false : data[InteractionEventDialogFields.EXTERNALIZATION_APPROVAL],
```

Comment:
You can use Boolean(data[InteractionEventDialogFields.EXTERNALIZATION_APPROVAL]) instead of checking if it's null and assigning false

Reply:
fixed